hazelcast/src/main/java/com/hazelcast/config/QueueConfig.java

```diff
@@ -24,14 +24,13 @@
 
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Comparator;
 import java.util.List;
 import java.util.Objects;
 
 import static com.hazelcast.internal.serialization.impl.SerializationUtil.readNullableList;
 import static com.hazelcast.internal.serialization.impl.SerializationUtil.writeNullableList;
-import static com.hazelcast.internal.util.Preconditions.checkAsyncBackupCount;
-import static com.hazelcast.internal.util.Preconditions.checkBackupCount;
-import static com.hazelcast.internal.util.Preconditions.checkNotNull;
+import static com.hazelcast.internal.util.Preconditions.*;
```

Comment:
Checkstyle doesn't allow star imports. I've created a patch file to fix all checkstyle issues and with some minor cleanup, feel free to apply it to your branch.
[Checkstyle_and_minor_cleanup.patch.zip](https://github.com/hazelcast/hazelcast/files/4780556/Checkstyle_and_minor_cleanup.patch.zip).

You can also run checkstyle and spotbugs locally - `mvn clean package -P checkstyle,spotbugs`.

If you want, you can also allow me to push commits to your branch and I can address such minor issues as I do the review.